jdi-light/src/main/java/com/epam/jdi/light/actions/ActionObject.java

```diff
@@ -70,13 +90,18 @@ private void resetElementTimeout() {
         if (element() != null)
             element().base().waitSec(elementTimeout);
     }
+
     public void setElementTimeout() {
         if (element() != null)
             element().base().waitSec(timeout());
     }
 
-    public JFunc1<Object, Object> overrideAction() { return overrideAction.get(); }
-    private CacheValue<JFunc1<Object, Object>> overrideAction = new CacheValue<>(this::getOverride);
+    public JFunc1<Object, Object> overrideAction() {
+        return overrideAction.get();
+    }
+
+    private final CacheValue<JFunc1<Object, Object>> overrideAction = new CacheValue<>(this::getOverride);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)